rawfile/config.py

```diff
@@ -0,0 +1,107 @@
+from pydantic_settings import (
+    BaseSettings,
+    CliSettingsSource,
+    CliSubCommand,
+    PydanticBaseSettingsSource,
+)
+from filesystem import FileSystemName
+from pydantic import (
+    AliasChoices,
+    AnyUrl,
+    BaseModel,
+    ByteSize,
+    StringConstraints,
+    DirectoryPath,
+    Field,
+    model_validator,
+)
+from typing import Annotated, Literal
+
+from utils.logs import LoggingFormats
+
+
+class CSIDriverCmd(BaseModel):
+    endpoint: AnyUrl
+    nodeid: str | None = None
+    metrics_port: int = Field(
+        validation_alias=AliasChoices("metrics-port", "METRICS_PORT"), default=9100
```

Comment:
Not familiar with pydantic, what do the alias do, is it the env variable name?

Reply:
without them, when validating model it will use `metrics_port` (or `METRICS_PORT`) and Cli arg will be `--metrics_port`

Reply:
https://docs.pydantic.dev/latest/concepts/pydantic_settings